### ISSUE TITLE
Missing tag in 2 actors: PFS season 1 cinder wolf (levels 1-2 and 3-4)

### ISSUE THREAD

**Issue by farabor** [foundryvtt/pf2e#9140]
Both level ranges are missing the knockdown trait on their jaws strike,  which per the pdf they have (and otherwise would make the knockdown ability they have unusable!)
